momoka-node/src/proofs/publications/comment/index.ts

```diff
@@ -1,86 +1,33 @@
-import { BigNumber } from 'ethers';
 import { LogFunctionType } from '../../../common/logger';
```

Comment:
I tried to keep changes to minimum but in the future we could consider moving the `checkDA(Post/Mirror/QuoteComment)` into the verifier class.